prime-router/src/main/kotlin/SubmissionsProvider.kt

```diff
@@ -0,0 +1,11 @@
+package gov.cdc.prime.router
```

Comment:
Some of these files are tiny, so you can combine some of these and reduce the number of files you have.  Also, your files are on the root folder, so try to organize them if you can.